test/e2e/apps/rc.go

```diff
@@ -67,10 +67,10 @@ var _ = SIGDescribe("ReplicationController", func() {
 	/*
 		Release : v1.13
 		Testname: Replication Controller, release pods
-		Description: Replication Controller Should release no longer matching pods, By creating a given Replicatio Controller. Pod should released, by using PollImmediate by checking the label of the pod .
+		Description: A Replication Controller (RC) is created, and its Pods are created. When the labels on one of the Pods change to no longer match the RC's label selector, the RC MUST release the Pod and update the Pod's owner references
 	*/
 
-	framework.ConformanceIt("should release no longer matching pods", func() {
+	framework.ConformanceIt("should release no longer matching pods [Conformance]", func() {
```

Comment:
Please remove the `[Conformance]` tag, this is what `ConformanceIt` does

Reply:
@spiffxp Removed [Conformance] label